### PR TITLE
Deleting tasks

### PR DESCRIPTION
**Pull request by Walter909**
Added an edit task pop Up and Can now delete tasks from homepage

### CODE REVIEW

**Review by mattriots — APPROVED** [review]
Looks good to me!

File: 2do_fe/src/redux/reducers/index.js [hunk 1]
```diff
@@ -24,13 +24,15 @@ const taskReducer = (state = initialState, action) => {
...
       return {
         ...state,
         loading: false,
         singleTask: payload,
       };
 
     case types.GET_ALL_TASKS_FAILURE:
+    case types.GET_SINGLE_TASKS_FAILURE:
     case types.ADD_TASKS_FAILURE:
       return { ...state, loading: false }; // error: payload
```
Comment (swlwin) [inline 2do_fe/src/redux/reducers/index.js:38]: Add reducer for Delete Task Request, Success and Failure as well. Other than that Delete function looks good to me!

**Review by swlwin — CHANGES_REQUESTED** [review]

File: 2do_fe/src/api.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import axios from "axios";
 
 const api = axios.create({
   baseURL: "http://localhost:5001",
-  // baseURL: "https://2do-307.azurewebsites.net/",
```
Comment (swlwin) [inline 2do_fe/src/api.js:5]: Try working on azure and see if that works as well. if not then it's okay to leave it as it is for now.

File: 2do_fe/src/redux/reducers/index.js (hunk 1 above)
Comment (Walter909) [reply 2do_fe/src/redux/reducers/index.js:41]: Changes made to fix bugs

File: 2do_fe/src/api.js (hunk 2 above)
Comment (Walter909) [reply 2do_fe/src/api.js:5]: I tried azure and there was some friction so I think for now we can leave it as

**Pull request merged**